spec/requests/general_info_spec.rb

```diff
@@ -67,7 +67,7 @@ def expect_form_failure
     expect(page).to have_field("Time", with: "1830")
   end
 
-  it "moves to the next page iff Save and Continue is clicked", driver: :poltergeist do
+  it "moves to the next page if Save and Continue is clicked", driver: :poltergeist do
```

Comment:
I actually did mean ["iff"](https://en.wikipedia.org/wiki/If_and_only_if) here and below, but feel free to change if it's confusing.